src/utils/tokens.ts

```diff
@@ -481,7 +483,8 @@ const tokens = {
    * @param {string} [options.mintAuthorityAddress] the address to send the mint authority created
    * @param {boolean} [options.createMelt=true] Whether to create a melt output
    * @param {string} [options.meltAuthorityAddress] the address to send the melt authority created
-   * @param {string|null} [options.nftData=null] NFT data to create an NFT token
+   * @param {string[]|null} [options.data=null] list of data strings to add each as a data script output
```

Comment:
question(if-minor): Do you think we should have the string be encoded?

For NFTs we used utf8 encoding since the data would be a string anyway, but now that we are allowing other types of data output should we have a different encoding, for instance base64 since its safe for utf8 strings and binary data.
Even if we keep using utf8 I think we should add the expected encoding on the docstrings, to make sure the caller knows how to pass the data.

Reply:
As discussed privately, we will keep the utf-8 encoding for now (added the info in the docs https://github.com/HathorNetwork/hathor-wallet-lib/pull/596/commits/8ed69441ff119f739a7f0437dd566184b2ec9532) and discuss this support in another issue (https://github.com/HathorNetwork/hathor-wallet-lib/issues/597)